src/client/eppo-client.ts

```diff
@@ -167,18 +163,20 @@ export default class EppoClient {
       this.banditModelConfigurationStore ?? null,
     );
 
-    this.requestPoller = initPoller(
-      POLL_INTERVAL_MS,
-      configurationRequestor.fetchAndStoreConfigurations.bind(configurationRequestor),
-      {
-        maxStartRetries: numInitialRequestRetries,
-        maxPollRetries: numPollRequestRetries,
-        pollAfterSuccessfulStart: pollAfterSuccessfulInitialization,
-        pollAfterFailedStart: pollAfterFailedInitialization,
-        errorOnFailedStart: throwOnFailedInitialization,
-        skipInitialPoll: skipInitialPoll,
-      },
-    );
+    const pollingCallback = async () => {
+      if (await this.flagConfigurationStore.isExpired()) {
```

Comment:
Polling = frequent checking of the cache and, if necessary, a fetch.

Reply:
Love it!